data/tribes/initialization/amazons/units.lua

```diff
@@ -1293,7 +1293,13 @@ wl.Descriptions():new_tribe {
             -- TRANSLATORS: Purpose helptext for an Amazon production site: Stonecutter's Hut
             purpose = pgettext("amazons_building", "Cuts raw pieces of granite out of rocks in the vicinity."),
             -- TRANSLATORS: Note helptext for an Amazon production site: Stonecutter's Hut
-            note = pgettext("amazons_building", "The stonecutter’s hut needs rocks to cut within the work area.")
+            note = pgettext("amazons_building", "The stonecutter’s hut needs rocks to cut within the work area."),
+            performance = {
+               -- TRANSLATORS: Performance helptext for an Amazon production site: Stonecutter's Hut, part 1
+               pgettext("amazons_building", "This building can produce one granite between %1$s and %2$s, depending on how far the stonecutter has stones."):bformat(format_seconds(52), format_minutes_seconds(1, 17)),
```

Comment:
```suggestion
               pgettext("amazons_building", "This building can produce one granite between %1$s and %2$s, depending on how far the stonecutter has to walk for stones."):bformat(format_seconds(52), format_minutes_seconds(1, 17)),
```
I think this is more appropriate in english language. so we should correct all instances like that, including the ones where @tothxa requested this part of wording.

Reply:
"depending on how far the stonecutter can find rocks"?

Reply:
Except "has" implies "within her/his work area", while "find" doesn't…

Reply:
as the difference is made by walking time I felt like emphasizing on walk so we could simply skip the "for resource" part and write "has to walk." in all instances.

Reply:
OTOH simply "has to walk" de-emphasizes that the player can affect this with careful placement of buildings and roads.

Reply:
I like the "has to walk." idea. New players can quickly learn where the workers have to walk, simply by watching game. The generic form is also appropriate for workers which go to random positions (like foresters, fishers).